src/egon/data/config.py

```diff
@@ -6,6 +6,7 @@
 
 from importlib_resources import files
 
+
```

Comment:
This style change is correct, but should be done in a separate commit. Ideally, style changes should be separate from changes modifying behaviour.